src/endpoints/createCategoriesEndpoint.js

```diff
@@ -37,20 +37,22 @@ export default (baseUrl: string): Endpoint<ParamsType, CategoriesMapModel> =>
             order: category.order,
             availableLanguages: mapAvailableLanguages(category.available_languages),
             parentPath: normalizePath(category.parent.path || basePath),
-            lastUpdate: moment.tz(category.modified_gmt, 'GMT')
+            lastUpdate: moment.tz(category.modified_gmt, 'GMT'),
+            hash: category.hash
           })
         })
 
       categories.push(new CategoryModel({
-        id: 0,
+        root: true,
         path: basePath,
         title: params.city,
         parentPath: '',
         content: '',
         thumbnail: '',
         order: -1,
         availableLanguages: new Map(),
-        lastUpdate: moment(0)
+        lastUpdate: moment(0),
+        hash: ''
```

Comment:
Do we have any problems if the hash is empty?

Reply:
I don't think so. The comparision of hashes should be a simple strcmp